### PR TITLE
PMM-12630 UI upgrade from 2.37.1 and lower does not work

### PR DESCRIPTION
**Pull request by adivinho**
PMM-12630

### CODE REVIEW

**Comment by adivinho** [comment]
It's a bug in el7. It can't update noarch to x86_64 package.
Screenshot 2023-10-30 at 10 23 58

**Review by BupycHuk — APPROVED** [review]

File: update/ansible/playbook/tasks/roles/clickhouse/tasks/main.yml [hunk 1]
```diff
@@ -33,6 +33,8 @@
       - percona-clickhouse-server
       - percona-clickhouse-common-static
       - percona-clickhouse-server-common
+      - clickhouse-client-21.3.20.1-2
```
Comment (ademidoff) [inline update/ansible/playbook/tasks/roles/clickhouse/tasks/main.yml:36]: What about uninstalling versions prior to 21.3.20? Or was this the only one used in PMM?

Comment (BupycHuk) [reply update/ansible/playbook/tasks/roles/clickhouse/tasks/main.yml:36]: I think a few lines above does that trick.

**Review by ademidoff — APPROVED** [review]

**Comment by BupycHuk** [comment]
merged the PR since it can be tested only on dev-latest

**Pull request merged**